modules/heat_conduction/test/tests/joule_heating/transient_jouleheating.i

```diff
@@ -61,28 +61,28 @@
   [./k]
     type = GenericConstantMaterial
     prop_names = 'thermal_conductivity'
-    prop_values = '0.95' #copper in cal/(cm sec C)
+    prop_values = '397.48' #copper in W/(m K)
     block = 0
   [../]
   [./cp]
     type = GenericConstantMaterial
     prop_names = 'specific_heat'
-    prop_values = '0.092' #copper in cal/(g C)
+    prop_values = '385.0' #copper in J/(kg K)
     block = 0
   [../]
   [./rho]
     type = GenericConstantMaterial
     prop_names = 'density'
-    prop_values = '8.92' #copper in g/(cm^3)
+    prop_values = '8920.0' #copper in kg/(m^3)
     block = 0
   [../]
   [./sigma]
     type = ElectricalConductivity
-    temp = T
-    ref_temp = 20
-    ref_resistivity = 0.0168
-    temp_coeff = 0.00386
-    length_scale = 1e-02
+    temperature = T
+    reference_temperature = 293.0       #in K
+    reference_resistivity = 1.68e-8     #copper in ohm-m
```

Comment:
Yes, this was changed for better numerical convergence.

Reply:
Using automatic scaling, I'm wondering if we might get away with getting rid of the `length_scale` parameter altogether now. In the test, I'm currently just setting it to "1" since my main length units are in meters, and it performs well.

Reply:
Why don't you set the default value for` length_scale ` as `1.0`.

Reply:
I'd also vote for removing the length scale parameter and using the automatic scaling capability. The `length_scale` parameter could be confusing to some users while the automatic scaling "comes" with MOOSE directly.....for what it's worth, I was one of the people that got confused by the length scale parameter

Reply:
@SudiptaBiswas I often see references operating at or quoting the lower length scales, so I opted to keep this in for the off-chance it was needed, purely for units conversion sake.

I did some basic testing - having both automatic scaling and a tiny default led to a converged problem, but non-realistic behavior for this set-point (extremely little heating, since the initial guess at each timestep was already practically "converged"), where removing the parameter (or having a default set to 1) led to the heating I would expect to see. So it seems easy to screw up, yet still see great convergence.

I'm still leaning toward removing the length scale parameter altogether, and modify the documentation to be a bit more units agnostic, since we really don't need this parameter for scaling purposes anymore with automatic scaling. That way, the default can be copper in `ohm-m`, but since `ElectricalConductivity` is rarely used by itself, users can square their resistivity units to the other units in their system on their own without an extra parameter they need to remember to tweak.

Reply:
Yeah sure, it can be removed. The user should provide a unit consistent value anyway.